feature/signupflow/signup/src/main/java/com/epmedu/animeal/signup/presentation/SignUpFlow.kt

```diff
@@ -28,4 +35,9 @@ fun SignUpFlow() {
         screen(SignUpRoute.EnterCode.name) { EnterCodeScreen() }
         screen(SignUpRoute.FinishProfile.name) { FinishProfileScreen() }
     }
+
+    if (state.isLoading) {
+        keyboardController?.hide()
+        ShimmerLoading(modifier = Modifier.fillMaxSize(), shape = RectangleShape, alpha = 0.7f)
```

Comment:
why use `ShimmerLoading` instead of the `FullScreenLoading` created on previous PR?

Reply:
I'm gonna replace it with `FullScreenLoading` in a new PR. This PR is only about managing loading state, not UI